.eslintrc

```diff
@@ -1,44 +1,46 @@
 {
-    "extends": ["prettier"],
-	"ecmaFeatures": {
-		"modules": true
+    parser: "@typescript-eslint/parser",
+    extends: [
+        "plugin:@typescript-eslint/recommended",
+        "prettier/@typescript-eslint",
+        "plugin:prettier/recommended",
+    ],
+    parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: "module"
+    },
+    env: {
+		es6: true,
+		browser: true
 	},
-	"parserOptions": {
-		"sourceType": "module",
-        "ecmaVersion": 2017
+    rules: {
```

Comment:
We use 2 spaces for indentation in other JS apps, no strong opinions though.
We tried to have shareable eslint/prettier configurations, but they ended up in the [monorepo](https://github.com/fastly/ux-platform), if any interest in sharing rules across projects, we should consider put these back to single repos.

Reply:
+1 for `"tabWidth": 2`